### PR TITLE
Localisation updates from https://translatewiki.net.

### PR DESCRIPTION
**Pull request by niedzielski**

### CODE REVIEW

File: Wikipedia/Localizations/azb.lproj/Localizable.strings [hunk 1]
```diff
@@ -45,35 +59,33 @@
 "search-titles" = "باشلیق‌لار";
 "search-recent-clear-cancel" = "وازگئچ";
 "search-recent-clear-delete-all" = "هامیسین سیل";
+"settings-title" = "تنظیملر";
+"settings-language" = "ویکی‌پدیا دیلی";
 "main-menu-title" = "آرتیق";
-"main-menu-language-title" = "ویکی‌پئدیادا $1 آختار";
```
Comment (niedzielski) [inline Wikipedia/Localizations/azb.lproj/Localizable.strings]: I wonder how this was passing previously.

Comment (montehurd) [inline Wikipedia/Localizations/azb.lproj/Localizable.strings]: @niedzielski that's just github's html for some reason showing the dollar sign on the right - I just opened the file with Atom and it's properly on the left.

The failing test is actually the "html in the translation" test :)

**Comment by montehurd** [comment]
👍

**Comment by joewalsh** [comment]
👍

**Pull request merged**